app/serializers/decidim/participatory_documents/suggestion_serializer.rb

```diff
@@ -47,6 +48,10 @@ def submitted_on(suggestion)
       def suggestion_body(suggestion)
         truncate(translated_attribute(suggestion.body), length: 50)
       end
+
+      def answer_text(suggestion)
+        truncate(translated_attribute(suggestion.answer), length: 50)
```

Comment:
```suggestion
        truncate(translated_attribute(suggestion.answer), length: Decidim::ParticipatoryDocuments.max_export_text_length)
```
I'd like to have this `50` number configurable. And, also if it is `zero`, then don't truncate at all the export.

Reply:
Thank you for review. Done